1-JavaScript/Week2/assignment/ex3-tellFortune.js

```diff
@@ -32,29 +32,28 @@ body, this code is now written once only in a separated function.
 
 // This function should take an array as its parameter and return
 // a randomly selected element as its return value.
-function selectRandomly(/* TODO parameter(s) go here */) {
-  // TODO complete this function
+function selectRandomly(value) {
+  const random = Math.floor(Math.random() * 5);
```

Comment:
```suggestion
  const random = Math.floor(Math.random() * value.length);
```
Suggestion:
Instead of hardcoding the number, it's better to use the array's length for more maintainable code.